### ISSUE TITLE
Duplicate callback triggers for AppointmentTimeChanged event in Outlook add-in

### ISSUE THREAD

**Issue by supermanklk** [OfficeDev/office-js#3396]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

**Environment**
Platform: MacBook Pro (16-inch, 2019)
Host: new - Outlook
Office version number: Microsoft® Outlook for Mac version： 16.73.2 (23052700)
Operating System: macOS 12.5

**Expected behavior**
I expected the Office.context.mailbox.item.addHandlerAsync function to trigger the callback function only once when modifying the date and time of an appointment in Outlook.

**Current behavior**
Instead of triggering the callback function once, the Office.context.mailbox.item.addHandlerAsync function is triggering the callback function twice when modifying the date and time of an appointment in Outlook. The first callback contains incorrect date information, while the second callback contains the correct information.

**Steps to reproduce**
Open Outlook on your Mac.
Create a new appointment or open an existing one.
Modify the date and time of the appointment.
Observe that the Office.EventType.AppointmentTimeChanged event handler is triggered twice, with the first callback containing incorrect date information and the second callback containing the correct information.

**Context**
This issue affects me as I'm developing an add-in for Outlook that relies on the AppointmentTimeChanged event. The duplicate callback triggers are causing unexpected behavior in my add-in, and I need assistance in resolving this issue.

**Comment by supermanklk** [comment]
`    Office.context.mailbox.item.addHandlerAsync(Office.EventType.AppointmentTimeChanged, function (args) {
      // 该代码块存在缺陷，原因是在mac-新版-outlook里面，触发一次日期的改变，会收到2次监听回调，第一次日期有问题，第二次没有问题，为了兼容该问题需要进行防抖处理

// The reason why this code block has defects is that in Mac-new version-Outlook, when a date change is triggered, two listening callbacks will be received. The first date has problems, and the second one has no problems. In order to be compatible with this problem, anti-buffing processing is needed

    });`

**Comment by supermanklk** [comment]
`
 Office.context.mailbox.item.addHandlerAsync(Office.EventType.AppointmentTimeChanged, function (args) {
// 该代码块存在缺陷，原因是在mac-新版-outlook里面，触发一次日期的改变，会收到2次监听回调，第一次日期有问题，第二次没有问题，为了兼容该问题需要进行防抖处理

// The reason why this code block has defects is that in Mac-new version-Outlook, when a date change is triggered, two listening callbacks will be received. The first date has problems, and the second one has no problems. In order to be compatible with this problem, anti-buffing processing is needed
`

**Comment by supermanklk** [comment]
`Office.context.mailbox.item.addHandlerAsync(Office.EventType.AppointmentTimeChanged, function (args) {
// 该代码块存在缺陷，原因是在mac-新版-outlook里面，触发一次日期的改变，会收到2次监听回调，第一次日期有问题，第二次没有问题，为了兼容该问题需要进行防抖处理 // The reason why this code block has defects is that in Mac-new version-Outlook, when a date change is triggered, two listening callbacks will be received. The first date has problems, and the second one has no problems. In order to be compatible with this problem, anti-buffing processing is needed`

**Comment by supermanklk** [comment]
`Office.context.mailbox.item.addHandlerAsync(Office.EventType.AppointmentTimeChanged, function (args) { }`

// The reason why this code block has defects is that in Mac-new version-Outlook, when a date change is triggered, two listening callbacks will be received. The first date has problems, and the second one has no problems. In order to be compatible with this problem, anti-buffing processing is needed

**Comment by exextoc** [comment]
@supermanklk Thanks for reporting this issue regarding AppointmentTimeChanged event. It has been put on our backlog. We unfortunately have no timelines to share at this point.

Internal tracking id: Office: 7967681
